src/storage/src/wal.rs

```diff
@@ -99,7 +105,7 @@ impl<S: LogStore> Wal<S> {
         seq: SequenceNumber,
         mut header: WalHeader,
         payload: Option<&Payload>,
-    ) -> Result<(u64, usize)> {
+    ) -> Result<u64> {
```

Comment:
```suggestion
    ) -> Result<Id> {
```
Prefer the aliased type